### PR TITLE
Feature 5: add support for device location history endpoint

### PR DESCRIPTION
**Pull request by azarc3**
Added support for endpoint.

### CODE REVIEW

File: ATTM2X/ATTM2X.Tests/CommandApiSpecs.cs [hunk 1]
```diff
@@ -628,5 +628,56 @@ public class Status_Counts
 		public int pending { get; set; }
 	}
 
-	#endregion " Response Objects "
+		public class Device
```
Comment (fsaravia) [inline ATTM2X/ATTM2X.Tests/CommandApiSpecs.cs]: This line looks oddly indented, is that correct?

File: ATTM2X/ATTM2X.Tests/DeviceApiSpecs.cs [hunk 2]
```diff
@@ -1041,22 +1044,23 @@ private static void DestroyTestDevices()
 			}
 		}
 
-		[TestMethod, Ignore] // todo: remove the Ignore attribute when the Device.LocationHistory() method has been added.
+		[TestMethod] // todo: remove the Ignore attribute when the Device.LocationHistory() method has been added.
```
Comment (fsaravia) [inline ATTM2X/ATTM2X.Tests/DeviceApiSpecs.cs]: Here you are removing the `Ignore` but leaving the `todo`, is that intended?

**Comment by fsaravia** [comment]
👍

File: ATTM2X/ATTM2X.Tests/DeviceApiSpecs.cs (hunk 2 above)
Comment (azarc3) [inline ATTM2X/ATTM2X.Tests/DeviceApiSpecs.cs]: Yeah, that was tomfool-ery on my part. The todo is removed in #17.

File: ATTM2X/ATTM2X.Tests/CommandApiSpecs.cs (hunk 1 above)
Comment (azarc3) [inline ATTM2X/ATTM2X.Tests/CommandApiSpecs.cs]: I didn't notice anything in my local copy. I'll be sure to watch out for that one when I pull the code next time. Either way though, it's just white space so if it's there then I'll be sure to fix it in a future PR.

**Pull request merged**